Tests/test_fslViewer.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+import unittest
+import viewer
+import sys
+import os
+import shutil
+import glob
+
+class fsl_con_f(unittest.TestCase): #Class for fsl_con_f tests
+	
+	def setUp(self): #Open necessary file
+		self.myString = ""
+		self.data = "data\\fsl_con_f.nidm.ttlTestResults\\postStats.html"
```

Comment:
@pwilliams1: could you use `os.path.join` as described [here](https://docs.python.org/2/library/os.path.html#os.path.join) rather than backslashes? This will ensure that the code works with different OS (Windows, Unix...).